jupyter-nextflow/nextflow-welcome.html

```diff
@@ -39,17 +39,21 @@ <h1 class=header>Welcome to the Nextflow Workspace</h1>
   </ul>
   <h2 class=header>Get started with Nextflow</h2>
   <p>If you are new to Nextflow, visit <a
-      href=https://www.nextflow.io>nextflow.io</a> for detailed information.
+      href=https://www.nextflow.io>nextflow.io</a> for detailed information. 
+    You can also learn more about using Nextflow in BRH Workspace in the <a
+      href=https://uc-cdis.github.io/BRH-documentation/01-home/>BRH Documentation</a>. 
```

Comment:
might as well link directly to the nextflow docs at https://uc-cdis.github.io/BRH-documentation/nextflow-getting-started/

Reply:
Because i don't want to necessarily link to the getting started page, because it's all about getting access to the workspace, which they already have. when I remake the docs, i can make a nextflow landing page that i link to - but for now, better to link to the brh documentation so they look in the nav menu and see nextflow